.github/fixtures/test-github-integration/cliff.toml

```diff
@@ -31,21 +30,22 @@ body = """
   {% raw %}\n{% endraw %}
 {% endif %}
 """
-# remove the leading and trailing whitespace from the template
-trim = true
-# changelog footer
-footer = """
+# Whether to remove leading and trailing whitespaces from all lines of the changelog's body.
+trim_body_whitespace = true
+# A Tera template to be rendered as the changelog's footer (see https://keats.github.io/tera/docs/#introduction).
+footer_template = """
 <!-- generated by git-cliff -->
 """
 
-[git]
-# parse the commits based on https://www.conventionalcommits.org
-conventional_commits = false
-# filter out the commits that are not conventional
-filter_unconventional = true
-# process each line of a commit as an individual commit
-split_commits = false
-# regex for preprocessing the commit messages
-commit_preprocessors = [{ pattern = '\((\w+\s)?#([0-9]+)\)', replace = "" }]
-# protect breaking changes from being skipped due to matching a skipping commit_parser
-protect_breaking_commits = false
+[commit]
+# Whether to parse commits according to the conventional commits specification.
+# Sets the commits' `group` (= `type`), `scope`, `message` (= `description`), `body`, `breaking`, `breaking_description` and `footers`.
+parse_conventional_commits = false
```

Comment:
I don't think `parse_` adds much more clarity here, so imo we could also avoid adding it to spare this breaking change.
But this comes to personal taste :)

Reply:
+1 for the name, but this option feels weirdly interlinked with `exclude_unconventional_commits`, a single option like `conventional_commits=<strict|enabled|disabled>` where `strict` means "also exclude unconventional" seems more intuitive

Reply:
I like the enum option, but we'd need a 4th option for requiring all commits to be unconventional in the future. I think `strict` would be more suitable for that 4th option. But that means we need another name for "exclude unconventional".